documentstore_migracao/processing/packing.py

```diff
@@ -78,67 +110,91 @@ def update_bar(pbar=pbar):
         )
 
 
-def download_asset(old_path, new_fname, dest_path):
-    """Returns msg, if error"""
+def get_asset(old_path, new_fname, dest_path):
+    """Obtém os ativos digitais no sistema de arquivo e realiza a persistência
+    no ``dest_path``.
+
+    Args:
+        old_path: Caminho do ativo
+        new_fname: Novo nome para o ativo
+        dest_path: Pasta de destino
+
+    Retornos:
+        Sem retornos.
+
+        Persiste o ativo no ``dest_path``
+
+    Exceções:
+        IOError
+        TypeError
+    """
     if old_path.startswith("http"):
-        location = old_path
+        asset_path = urlparse(old_path).path
     else:
-        try:
-            location = urljoin(config.get("STATIC_URL_FILE"), old_path.strip())
-        except ValueError as exc:
-            return 'cannot join URL parts "%s" and "%s": %s' % (
-                config.get("STATIC_URL_FILE"),
-                old_path,
-                exc,
-            )
+        asset_path = old_path
+
+    asset_path = asset_path.strip('/')
 
     # Verifica se o arquivo ja foi baixado anteriormente
     filename_m, ext_m = files.extract_filename_ext_by_path(old_path)
     dest_path_file = os.path.join(dest_path, "%s%s" % (new_fname.strip(), ext_m))
     if os.path.exists(dest_path_file):
-        logger.debug("Arquivo ja baixado: %s", dest_path_file)
+        logger.debug("Arquivo já armazenado na pasta de destino: %s", dest_path_file)
         return
 
     try:
-        request_file = request.get(location, timeout=int(config.get("TIMEOUT") or 10))
-    except request.HTTPGetError as e:
+        source_path = config.get('SOURCE_PDF_FILE') if ext_m.endswith('pdf') else config.get('SOURCE_IMG_FILE')
```

Comment:
é melhor testar a existência nas duas pastas `pdf` e `img` porque se ele for um material suplementar talvez esteja em `img`...

Reply:
@robertatakenaka 

Segui a lógica de que se é um `PDF` está na pasta de `PDF`, o resto está em img dessa forma incluo o material suplementar.

Reply:
@jamilatta pode ser que alguns materiais suplementares sejam `pdf`, dessa forma eles estariam na pasta de imagens. Da forma que está o código comporta esse requisito?

Reply:
Não. 

Para acredito ser necessário avaliar o primeiro seguimento do caminho para o arquivo, o que acha?

Reply:
Não tenho uma resposta para isso mas suspeito que esse comportamento esteja implementado no comando que empacota os documentos nativos, talvez seja adequado se inspirar do que é feito lá.

Reply:
@joffilyfe 

Vou dar uma olhada!

Reply:
@scieloorg/scielo-ps-developers 

Ajustado! 

Tirei uma listagem dos arquivos em pdf presente na pasta de `revistas/img`, vejam: 

[Uploading pdf_on_img_folder.txt…]()

Reply:
Posso estar enganado, pois, pode ser que exista algum formato de nome de `pdf` que desconheço, mas não encontrei qualquer `pdf` que seja suplemento na pasta de `revistas/img`

A primeira linha do arquivo aponta para o seguinte`pdf`: `revistas/img/hcsm/v11s1/html/hcsm/v11s1/05.pdf`

Porém, ao olhar o fascículo é possível ver que os `pdfs` também estão em: `/pdf/hcsm/v11s1/05.pdf` 
https://www.scielo.br/scielo.php?script=sci_issuetoc&pid=0104-597020040004&lng=en&nrm=iso/

Reply:
Se eu não estiver errado o suplemento do `pdf` tem sufixo `-s`, correto @robertatakenaka ?

Reply:
@jamilatta  me lembrei que em alguns casos o pessoal colocava alguns arquivos em `/htdocs/img/revistas/acron/volnum/html/*`

Reply:
> Se eu não estiver errado o suplemento do `pdf` tem sufixo `-s`, correto @robertatakenaka ?

não necessariamente